### PR TITLE
Bump key-share version and add its changelog

### PR DESCRIPTION
**Pull request by maurges**
Should fix releasing keygen and cggmp21

### CODE REVIEW

File: key-share/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## v0.2.0
+* Add support of HD wallets compatible with BIP-32 and SLIP-10 [#68],
+  [#74], [#75]
+* Prohibit key shares with zero secret share or secret key [#82]
```
Comment (survived) [inline key-share/CHANGELOG.md:6]: Should we remove the same log messages from cggmp21/changelog ?

Comment (maurges) [reply key-share/CHANGELOG.md:6]: It touched cggmp21 in that we had to alter it a bit to support it, so I think it should be in both

Comment (survived) [reply key-share/CHANGELOG.md:6]: Well, in the future, I don't think we'll be duplicating changelog messages between the logs

Comment (survived) [reply key-share/CHANGELOG.md:6]: we're too lazy for that

**Review by survived — APPROVED** [review]

**Pull request merged**